frontend/packages/dev-console/integration-tests/features/monitoring/dashboard-display.feature

```diff
@@ -1,67 +1,59 @@
 @monitoring
-Feature: Monitoring Page
-              As a user, I should be able to perform actions related to monitoring a project and deployments inside it
+Feature: Observe Page
+              As a user, I should be able to perform actions related to observe a project and deployments inside it
 
 
         Background:
             Given user is at developer perspective
-              And user has created or selected namespace "aut-monitoring-1"
+              And user has created or selected namespace "aut-observe-1"
 
 
         @smoke
-        Scenario: Monitoring Page: M-02-TC01
+        Scenario: Observe Page: M-02-TC01
             Given user is at Add page
-             When user navigates to Monitoring page
+             When user navigates to Observe page
              Then user will see Dashboard, Metrics, Alerts, Events tabs
 
 
         @regression
-        Scenario: Dashboard tab on the Monitoring page for particular workload: M-02-TC02
+        Scenario: Dashboard tab on the Observe page for particular workload: M-02-TC02
             Given user is at Add page
               And user has created workload "parks-test" with resource type "Deployment"
               And user opened the url of the workload "parks-test" in topology page
-              And user is on Monitoring page
+              And user is on Observe page
              When user clicks on "Dashboard" tab
               And user selects the workload "parks-test" from the dropdown
-             Then user will see the CPU Usage on Dashboard tab
-              And user will see the Memory Usage on Dashboard tab
-              And user will see Receive Bandwidth on Dashboard tab
-              And user will see Transmit Bandwidth on Dashboard tab
-              And user will see the Rate of Received Packets on Dashboard tab
-              And user will see the Rate of Transmitted Packets on Dashboard tab
-              And user will see the Rate of Received Packets Dropped on Dashboard tab
-              And user will see the Rate of Transmitted Packets Dropped on Dashboard tab
+             Then user will see the "Namespace (Workloads)" on Dashboard tab
+              And user will see the "Namespace (Pods)" on Dashboard tab
+              And user will see "Workloads" on Dashboard tab
+              And user will see "Pods" on Dashboard tab
```

Comment:
This scenario is changed. Now user can select the workload after selecting the `Kubernetes / Compute Resources / workload` dashboard.

Reply:
Updated